tests/unit_tests/test_cli.py

```diff
@@ -1281,3 +1282,48 @@ def test_python_env_output_formatting():
         """)
 
     _assert_matching_formatting(OutputFormat.FULL, empty_python_env, full)
+
+
+@pytest.mark.parametrize("output_flag", [True, False])
+@pytest.mark.parametrize("update", [True, False])
+@patch("blueapi.cli.cli.config.CONFIG_SCHEMA_LOCATION")
+def test_config_schema(
+    config_schema_location_mock: Mock,
+    runner: CliRunner,
+    output_flag: bool,
+    update: bool,
+    tmp_path: Path,
+):
+    args = ["config-schema"]
+
+    tmp_path = tmp_path / "foo.json"
+
+    if output_flag:
+        # args.append(f"-o {output}")
+        args.append("-o")
+        args.append(f"{tmp_path}")
+
+    if update:
+        args.append("-u")
+
+    result = runner.invoke(
+        main,
+        args,
+    )
+
+    if output_flag and (not update):
+        with tmp_path.open("r") as stream:
+            assert (
+                DeepDiff(json.load(stream), ApplicationConfig.model_json_schema()) == {}
+            )
+
+    elif update:
+        config_schema_location_mock.open.assert_called()
+        with config_schema_location_mock.open() as stream:
+            stream.write.assert_called()
+
+    else:
+        assert (
+            DeepDiff(json.loads(result.output), ApplicationConfig.model_json_schema())
+            == {}
+        )
```

Comment:
```suggestion
    expected = ApplicationConfig.model_json_schema()

    if output_flag and (not update):
        with tmp_path.open("r") as stream:
            assert json.load(stream) == expected

    elif update:
        config_schema_location_mock.open.assert_called()
        with config_schema_location_mock.open() as stream:
            stream.write.assert_called()

    else:
            assert json.load(result.output) == expected
```

This works when I run it locally and prevents importing DeepDiff

Reply:
LGTM, but I think line 1324 should be `json.loads`